db/migrations/postgres/000070_add_subscriptions_filters.up.sql

```diff
@@ -0,0 +1,7 @@
+BEGIN;
+ALTER TABLE subscriptions ADD COLUMN filters TEXT;
```

Comment:
We do need to migrate existing filters. You may be able to build the JSON yourself with something like:
```
UPDATE subscriptions SET filters='{"events":"' || filter_events || '","topics":"' || filter_topics || '"}';
```
^ totally untested, but maybe a starting point to try out.